mods/linked/moves.js

```diff
@@ -0,0 +1,462 @@
+					for (const moveSlot of source.moveSlots) {
+						if (moveSlot.id === lastMove.id) {
+							moveSlot.pp = 0;
+							this.add('-activate', source, 'move: Grudge', this.getMove(lastMove.id).name);
+						}
+					}
+				}
+			},
+			onBeforeMovePriority: 100,
+			onBeforeMove: function (pokemon) {
+				if (pokemon.moveThisTurn) return; // Second stage of a Linked move
+				this.debug('removing Grudge before attack');
+				pokemon.removeVolatile('grudge');
+			},
+		},
+	},
+	spite: {
+		inherit: true,
+		onHit: function (target) {
+			let lastMove = target.getLastMoveAbsolute();
+			if (lastMove && target.deductPP(lastMove.id, 4)) {
+				this.add("-activate", target, 'move: Spite', this.getMove(lastMove.id).name, 4);
+				return;
+			}
+			return false;
+		},
+	},
+
+	/**
+	 * Other moves that check `pokemon.lastMove`
+	 * (may behave counter-intuitively if left unmodded)
+	 *
+	 **/
+
+	conversion2: {
+		inherit: true,
+		onHit: function (target, source) {
+			let lastMove = target.getLastMoveAbsolute();
+			if (!lastMove) return false;
+			let possibleTypes = [];
+			let attackType = lastMove.type;
+			for (let type in this.data.TypeChart) {
+				if (source.hasType(type) || target.hasType(type)) continue;
+				let typeCheck = this.data.TypeChart[type].damageTaken[attackType];
+				if (typeCheck === 2 || typeCheck === 3) {
+					possibleTypes.push(type);
+				}
+			}
+			if (!possibleTypes.length) {
+				return false;
+			}
+			let randomType = this.sample(possibleTypes);
+
+			if (!source.setType(randomType)) return false;
+			this.add('-start', source, 'typechange', randomType);
+		},
+	},
+	destinybond: {
+		inherit: true,
+		effect: {
+			onStart: function (pokemon) {
+				this.add('-singlemove', pokemon, 'Destiny Bond');
+			},
+			onFaint: function (target, source, effect) {
+				if (!source || !effect || target.side === source.side) return;
+				if (effect.effectType === 'Move' && !effect.isFutureMove) {
+					this.add('-activate', target, 'move: Destiny Bond');
+					source.faint();
+				}
+			},
+			onBeforeMovePriority: -1,
+			onBeforeMove: function (pokemon, target, move) {
+				// Second stage of a Linked move does not remove Destiny Bond
+				if (pokemon.moveThisTurn || move.id === 'destinybond') return;
+				this.debug('removing Destiny Bond before attack');
+				pokemon.removeVolatile('destinybond');
+			},
+			onMoveAborted: function (pokemon, target, move) {
+				pokemon.removeVolatile('destinybond');
+			},
+			onBeforeSwitchOutPriority: 1,
+			onBeforeSwitchOut: function (pokemon) {
+				pokemon.removeVolatile('destinybond');
+			},
+		},
+	},
+	trumpcard: {
+		inherit: true,
+		basePowerCallback: function (pokemon) {
+			let move = pokemon.getMoveData(pokemon.getLastMoveAbsolute()); // Account for calling Trump Card via other moves
+			switch (move.pp) {
+			case 0:
+				return 200;
+			case 1:
+				return 80;
+			case 2:
+				return 60;
+			case 3:
+				return 50;
+			default:
+				return 40;
+			}
+		},
+	},
+};
```

Comment:
Is Fusion Bolt + Fusion Flare no longer supported?
Or does it work correctly with default mechanics now?

Reply:
it works correctly

Reply:
just kidding, after testing, they dont work